### PR TITLE
Remove /opt bits - so my entire /opt doesn't become readonly

### PR DESCRIPTION
**Pull request by goochjj**
Remove /opt bits - so my entire /opt doesn't become readonly
Drop the includes, and cmake stuff, and things flatcar runtime doesn't need.
Tweak the README.md.

### CODE REVIEW

**Comment by jepio** [comment]
Sorry, was gone for a couple of weeks.

Happy to merge this if it works for you. Do I understand correctly that the torcx package is meant to be used together with the sysext?

**Review by jepio — APPROVED** [review]

**Comment by goochjj** [comment]
@jepio No worries.  The problem I ran into with just the sysext was that docker still tries to call docker.  If you don't use the torcx, you could use docker too - and have a dual setup.

In my case, I want `docker` to call `podman`, so my options as I see it are
1) Expand the sysext to also replace the /usr/bin/docker file

The problem with this method is other things, like runc, containerd, containerd-shim, etc are symlinked to /usr/bin/docker, so replacing it with something other than a torcx redirector would cause other issues. (And I didn't want to replace ALL the binaries, that seemed dumb)

2) roll my own replacement docker-torcx package, that only provides a docker binary (which calls podman)... So /usr/bin/docker finds bin/docker in the torcx package which calls podman.  dockerd, runc, etc just say "this torcx package doesn't contain a runc binary" or whatever, which is also good.


I've since expanded the torcx package (which should, perhaps, be a separate repo entirely... it is in my CI) to borrow the /lib/systemd/network/ files that make *veth* interfaces and br-* (and now cni-*) interfaces not be managed by networkd...

It just seemed like the most elegant way to solve it.  As long as the torcx shims are there in /usr/bin, solving this with torcx makes sense.  If docker moves from torcx to a sysext in the future, (in a way that I can remove), then I'd solve that a different way. (put a docker wrapper in the podman sysext)  But I'm not sure putting /usr/lib/systemd/network/ files in a sysext is the right way at this point, it might be. /shrug/

We can have this conversation elsewhere if you want.

TL;DR - you only need the torcx if you want to remove docker from your system, and you want `docker` to call `podman`
